client/src/app/+accounts/account-video-channels/account-video-channels.component.html

```diff
@@ -8,7 +9,7 @@
         <a [routerLink]="getVideoChannelLink(videoChannel)" i18n-title title="See this video channel">
           <img [src]="videoChannel.avatarUrl" alt="Avatar" />
 
-          <div>{{ videoChannel.displayName }}</div>
+          <h2 class="section-title">{{ videoChannel.displayName }}</h2>
```

Comment:
In that instance, `section-title` is adding a `border-top`, which should only be applied to the parent container.

Reply:
I fixed it in [https://github.com/Chocobozzz/PeerTube/pull/2871/commits/63c702b065e2b3bca981a3cc51035c285bc74e0a ](https://github.com/Chocobozzz/PeerTube/pull/2871/commits/63c702b065e2b3bca981a3cc51035c285bc74e0a).